lib/rmt/cli/mirror.rb

```diff
@@ -0,0 +1,33 @@
+module RMT::CLI::Mirror
+
+  def self.mirror
+    require 'rmt/mirror'
+    require 'rmt/config'
+
+    repositories = Repository.where(mirroring_enabled: true)
+
+    if repositories.empty?
+      warn 'There are no repositories marked for mirroring.'
+      return
+    end
+
+    repositories.each do |repository|
+      begin
+        RMT::Mirror.new(
+          mirroring_base_dir: Settings.mirroring.base_dir,
+          mirror_src: Settings.mirroring.mirror_src,
+          repository_url: repository.external_url,
+          local_path: repository.local_path,
+          auth_token: repository.auth_token,
+          logger: Logger.new(STDOUT)
+        ).mirror
+
+        repository.last_mirrored_at = DateTime.now.utc
+        repository.save!
```

Comment:
Database operations outside of model.

Reply:
@wstephenson what are your thoughts on this?

Reply:
Could wrap it in a simple model method.

Reply:
Could -- yes. Should I, though?
Same question the rest of the DB operations.

Reply:
> Should I, though?

Yes! :neckbeard:

Reply:
Okey-doke! :grimacing::+1: